exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
+var name = "Niamh";
+
+var nameLength = name.length;
+
+console.log(nameLength)
```

Comment:
well done on getting the length of the name, you missed part of the exercise where you concatenate the string and use the `trim()` method 🙂